cmake/modules/platform/toolcfg/xlc.cmake

```diff
@@ -212,8 +212,8 @@ else()
 
 	function(_omr_toolchain_separate_debug_symbols tgt)
 		set(exe_file "$<TARGET_FILE:${tgt}>")
-		omr_get_target_path(target_path ${tgt})
-		omr_replace_suffix(dbg_file "${target_path}" ".debuginfo")
+		omr_get_target_output_genex(${tgt} output_name)
```

Comment:
The function is `omr_get_target_output_info`, not `omr_get_target_output_genex`.

Reply:
i have have changes to OmrUtility which i forgot to commit (which renames the function and changes the signature), however I am holding off updating this pr until I get the prefix issue resolved